src/main/java/marquez/api/JobRun.java

```diff
@@ -73,6 +76,10 @@ public Integer getCurrentState() {
     return currentState;
   }
 
+  public UUID getJobVersionGuid() {
+    return jobVersionGuid;
+  }
+
   @Override
   public boolean equals(final Object o) {
     if (this == o) return true;
```

Comment:
Did we want to clean up this class to us lombok? I think we can remove some of the logic around state transitions in a separate PR?

Reply:
I left the API models the way they were since Aleks will be addressing in his resource layer PR

Reply:
I'll revert this change, probably shouldn't have touched this file

Reply:
👍